### PR TITLE
Fixing issues

### PR DESCRIPTION
**Pull request by upatricck**
Fixing issues

### CODE REVIEW

File: seshdash/views.py [hunk 1]
```diff
@@ -996,6 +996,15 @@ def site_add_edit(request):
     return render(request, 'seshdash/settings/site_settings.html', context_dict)
 
 
+@login_required
+def delete_site(request, site_id):
+    """
+    Deletes a sesh site
+    """
+    site = Sesh_Site.objects.filter(id=site_id).first()
```
Comment (jlark) [inline seshdash/views.py]: How do we know the user has permission to delete the site? There needs to be a sort of permission check here.

**Comment by upatricck** [comment]
screenshot from 2017-01-30 10-46-38

**Comment by upatricck** [comment]
I think we will also need a page to show all the alerts, we can not fit the all in the alert pane.

**Comment by jlark** [comment]
OK, we can show the last 10 or 20 and add a link on the bottom that says show all. Which can link to an alert log page.

**Comment by jlark** [comment]
👍

**Pull request merged**